README.md

```diff
@@ -1,30 +1,59 @@
 # Daphne
 
-## Integration tests
+Daphne is a Rust implementation of the Distributed Aggregation Protocol
+([DAP])(https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap) standard. DAP is
+under active development in the PPM working group of the IETF.
 
-[miniflare](https://miniflare.dev/) is used to test integration of Daphne-Worker
-with Daphne. These tests can be run from Docker:
+This software is intended to support experimental DAP deployments and is not yet
+suitable for use in production. Daphne will evolve along with the DAP draft:
+Backwards compatibility with previous drafts won't be guaranteed until the draft
+itself begins to stabilize. API-breaking changes between releases should also be
+expected.
+
```

Comment:
maybe just mention how version number is related to the draft.

Reply:
I don't want to promise anything here until we get to draft-01.